pages/en/contrib/code-contrib.md

```diff
@@ -34,18 +34,14 @@ Another place to look is to find our issues on GitHub with `good first issue` ta
 
 ### LoopBack versions
 
-LoopBack maintains a current version, a long-term support (LTS) version, and a maintenance version.  New features go into the current version; bug fixes can go into the LTS and maintenance versions, depending on the nature of the bug.  For more information, see [Long-term support](Long-term-support.html).
+LoopBack maintains a current version, a long-term support (LTS) version, and a maintenance version.  New features go into the current version; bug fixes can go into the active LTS and maintenance LTS versions, depending on the nature of the bug.  For more information, see [Long-term support](Long-term-support.html).
```

Comment:
nitpick: 
...**an active** long-term support (LTS) version, and a maintenance **LTS** version...

Reply:
hm. I think the previous sentence mentioned what LTS stands for, and it's not needed for the subsequent sentence.